roles/openshift_rhv/tasks/build_vm_list.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Creates a dictionary for use with oVirt.vm-infra role
+# https://github.com/oVirt/ovirt-ansible-vm-infra
+- name: Create virtual machine list fact
+  set_fact:
+    vms: >-
+      {{ vms|default([]) }} + [
+      {% for iter in range(item.count) -%}
+      {% if iter > 0 -%},{% endif -%}
+      {
+      'name': '{{ item.name }}{{ iter }}.{{ public_hosted_zone }}',
```

Comment:
What var files do you use for this ? The public_hosted_zone in configured in the OpenShift variables file but not in the provisioning-vars.yaml.example

Reply:
Good catch. I'll need to better document that and ensure those vars are properly name-spaced to openshift_rhv.